Scripts/script-BuildEWSQuery.yml

```diff
@@ -2,7 +2,13 @@ commonfields:
   id: BuildEWSQuery
   version: -1
 name: BuildEWSQuery
+releaseNotes: "Add paramter for stripping the subject from prefixes"
 script: |-
+  import re
+
+  # Regex for removing forward/replay prefixes
+  p = re.compile('([\[\(] *)?(RE|FWD?) *([-:;)\]][ :;\])-]*|$)|\]+ *$', re.IGNORECASE)
```

Comment:
This regex seems to be removing data from middle of line as well, but i might be wrong here